postgres.md

```diff
@@ -208,34 +208,38 @@ CREATE TABLE foo(bar integer, NOT NULL (bar));
 CREATE TABLE foo(bar integer NOT NULL);
 ```
 
-##### `PRIMARY KEY` constraints must be declared into the table declaration WITH explicit column reference.
+##### Contraints must de declared outside of its referencing table declaration. This allows a much more malleable code.

```

Comment:
I just removed the rule. That's not a real coding style, but a constraint depending on your infrastructure needs as you mentioned!